resolvers.ts

```diff
@@ -490,6 +486,38 @@ export const resolvers: Resolvers = {
     workflowRuns: async (root, args, context) => {
       const input = args.input;
 
+      // If we provide a list of workflowRunIds, we assume that this is for getting valid consensus genome workflow runs.
+      // This endpoint only provides id, ownerUserId, and status.
+      if (input?.where?.id?._in) {
+        const body = {
+          workflowRunIds: input?.where?.id?._in.map(id => id && parseInt(id)),
+        };
+
+        const { workflowRuns } = await postWithCSRF(
+          `/workflow_runs/valid_consensus_genome_workflow_runs`,
+          body,
+          args,
+          context,
+        );
+        return workflowRuns.map(
+          (run): query_workflowRuns_items => ({
+            id: run.id,
+            ownerUserId: run.owner_user_id,
+            status: run.status,
+            entityInputs: {
+              edges: [
+                {
+                  node: {
+                    fieldName: "Sample",
+                    inputEntityId: run.sample,
+                  },
+                },
+              ],
+            },
```

Comment:
I think this is an addition to this resolver right?  I'm not sure if you just haven't gotten to it yet (since this PR is a draft), but can we add a unit test(s) for this case?